swarm/src/executor.rs

```diff
@@ -1,3 +1,4 @@
+//! Provides executors for spawning background tasks.
```

Comment:
Oversight on my end on https://github.com/libp2p/rust-libp2p/pull/3097: Would you mind adding a license header here? See other files in the repository.